packages/alea-frontend/pages/blog/new.tsx

```diff
@@ -26,17 +27,25 @@ const NewPostPage: NextPage = () => {
   const [body, setBody] = useState('');
   const postId = generatePostId(title);
 
+  if (title) localStore.setItem('blogTitle', title);
```

Comment:
Avoid depending on the render cycle for this.

Make the change as soon as you detect it. So move this code in the `onValueChange` function of the MystEditor component.